hazelcast/src/main/java/com/hazelcast/spi/tenantcontrol/TenantControl.java

```diff
@@ -43,18 +41,19 @@
     /**
      * Establish this tenant's thread-local context
      * Particular TenantControl implementation will control the details of how
```

Comment:
Minor - "Particular TenantControl implementation will control the details of how" can probably be rephrased as it sounds strange.

Reply:
done